docs/sqlalchemy.rst

```diff
@@ -330,44 +329,48 @@ characters::
 .. NOTE::
 
     The information we supply via the ``details`` column isn't defined in the
-    :ref:`original SQLAlchemy table definition <table-definition>`. These
-    details can be `specified`_ when you create the column in CrateDB, or you
-    can configure the column to support `dynamic values`_.
+    :ref:`original SQLAlchemy table definition <table-definition>` schema.
+    These details can be specified as *object column policy* when you create
+    the column in CrateDB, you can either use the :ref:`STRICT column policy
+    <crate-reference:type-object-columns-strict>`, or the :ref:`DYNAMIC column
+    policy <crate-reference:type-object-columns-dynamic>`.
 
 .. NOTE::
 
     Behind the scenes, if you update an ``Object`` property and ``commit`` that
-    change, the `UPDATE`_ statement sent to CrateDB will only include the data
-    necessary to update the changed subcolumns.
+    change, the :ref:`UPDATE <crate-reference:dml-updating-data>` statement sent
+    to CrateDB will only include the data necessary to update the changed
+    sub-columns.
 
 .. _objectarray:
 
 ``ObjectArray``
 ...............
 
 In addition to the `Object`_ type, the CrateDB SQLAlchemy dialect also provides
-a ``ObjectArray`` type, which is structured as a `list`_ of `dictionaries`_.
+a ``ObjectArray`` type, which is structured as a :class:`py:list` of
```

Comment:
Very nice to have those direct links! Do you know why one renders in bold and the other one doesn't?

<img width="846" alt="Screenshot 2022-12-27 at 13 02 01" src="https://user-images.githubusercontent.com/218003/209664020-cbf34109-2679-4024-91da-65540255a828.png">

Reply:
That's weird, thanks for spotting. I do not have an idea yet, why this is happening.

Reply:
### Observations I
```html
<a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#list" title="(in Python v3.11)">
  <code class="docutils literal notranslate">
    <span class="pre">list</span>
  </code>
</a>
```
```html
<a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#dict" title="(in Python v3.11)">
  <code class="xref py py-class docutils literal notranslate">
    <span class="pre">dictionaries</span>
  </code>
</a>
```

It looks like the `xref` class (from `basic.css`) adds the `font-weight: bold;` CSS directive.


### Observations II
Funny enough, when rendering locally (not on RTD), those additional `code` tags, including the corresponding classes, are not rendered at all. Instead, the HTML looks like this:
```html
<a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#lists">list</a>
<a class="reference external" href="https://docs.python.org/3/library/stdtypes.html#dict">dictionaries</a>
```

And visually, it looks like this:
![image](https://user-images.githubusercontent.com/453543/209686798-2a67d98a-dc56-420f-9ef7-7b030d8a4273.png)


### Next steps
So, I guess we will need to find out why the HTML output is different when being built on/by RTD.

Reply:
After resetting my local sandbox environment, I am also getting the same results as above, where the rendered HTML at this spot includes `<code>` tags, and where the same CSS classes `xref py py-class` are present on the `:class:dictionaries <py:dict>` link.

It looks like the custom label, `dictionaries`, is responsible for the additonal `xref py py-class` classes.

Reply:
I have not been able to discover the root cause, so I fixed it with https://github.com/crate/crate-docs-theme/pull/366.

Reply:
With the patch to `crate-docs-theme`, it now looks like this:

![image](https://user-images.githubusercontent.com/453543/209948733-2a55737d-4a93-4c1e-862e-4f24f7560237.png)